### PR TITLE
wpc-582 Add package version in the editor

### PR DESCRIPTION
**Pull request by tomaszdurka**

### CODE REVIEW

**Review by camilleory — DISMISSED** [review]

**Comment by tomaszdurka** [comment]
@elias-summermatter @michael-scheurer do you think that helps. or something more visible with api version too?

File: packages/api/src/context.ts [hunk 1]
```diff
@@ -447,6 +455,14 @@ export async function contextFromRequest(
       if (!updatedPayment) throw new Error('Error during updating payment') // TODO: this check needs to be removed
 
       return updatedPayment
+    },
+
+    async getSystemInformation(): Promise<SystemInformation> {
```
Comment (michael-scheurer) [inline packages/api/src/context.ts:460]: A function should always come with a minimum of comments. As a dev I want to know why this function and what it does.

File: packages/api/__tests__/api/private/index.ts [hunk 2]
```diff
@@ -656,6 +658,7 @@ export type MemberPlan = {
   slug: Scalars['String']
   image?: Maybe<Image>
   description?: Maybe<Scalars['RichText']>
+  tags?: Maybe<Array<Scalars['String']>>
```
Comment (elias-summermatter) [inline packages/api/__tests__/api/private/index.ts:661]: Why?

File: packages/api/__tests__/api/private/index.ts [hunk 3]
```diff
@@ -671,13 +674,15 @@ export type MemberPlanConnection = {
 export type MemberPlanFilter = {
   name?: Maybe<Scalars['String']>
   active?: Maybe<Scalars['Boolean']>
+  tags?: Maybe<Array<Scalars['String']>>
```
Comment (elias-summermatter) [inline packages/api/__tests__/api/private/index.ts:677]: Why?

File: packages/api/__tests__/api/private/index.ts [hunk 4]
```diff
@@ -671,13 +674,15 @@ export type MemberPlanConnection = {
...
 }
 
 export type MemberPlanInput = {
   name: Scalars['String']
   slug: Scalars['String']
   imageID?: Maybe<Scalars['ID']>
   description?: Maybe<Scalars['RichText']>
+  tags: Array<Scalars['String']>
```
Comment (elias-summermatter) [inline packages/api/__tests__/api/private/index.ts:685]: Why?

File: packages/editor/src/server/index.tsx [hunk 5]
```diff
@@ -49,6 +49,7 @@ async function asyncMain() {
           <meta name="viewport" content="width=device-width, initial-scale=1" />
           <link rel="icon" type="image/png" sizes="128x128" href="/static/favicon-128.png" />
           <link rel="icon" type="image/png" sizes="64x64" href="/static/favicon-64.png" />
+          <title>WePublish Editor (v{process.env.npm_package_version})</title>
```
Comment (elias-summermatter) [inline packages/editor/src/server/index.tsx:52]: This does not work as we tested together :-)

**Review by elias-summermatter — CHANGES_REQUESTED** [review]

**Review by michael-scheurer — COMMENTED** [review]
I think you're mixing up TikTok videos with system information. Why?

Secondly, to have the current version visible is nice to have. In my opinion it really doesn't solve the original issue from Jira which is described as "As a developer i want a clear release strategy".
A clear release strategy has to be implemented over organisation and not code ;)

File: packages/api/src/graphql/query.public.ts [hunk 6]
```diff
@@ -426,6 +427,14 @@ export const GraphQLPublicQuery = new GraphQLObjectType<undefined, Context>({
         const updatedInvoices = await dbAdapter.invoice.getInvoicesByUserID(user.id)
         return updatedInvoices.find(invoice => invoice !== null && invoice.id === id)
       }
+    },
+    getSystemInformation: {
```
Comment (michael-scheurer) [inline packages/api/src/graphql/query.public.ts:431]: This information should never be public. Avoid telling the world, which bugs you have in your application.

**Comment by tomaszdurka** [comment]
Closing for now.

**Pull request closed without merging**